src/components/schedule/DaySchedule.tsx

```diff
@@ -25,6 +33,7 @@ function DaySchedule({
     daySchedule: RezervoDaySchedule;
     selectedLocationIds: string[];
     selectedCategories: string[];
+    excludedClassTimeFilters: ExcludeClassTimeFilter[];
```

Comment:
```suggestion
    excludeClassTimeFilters: ExcludeClassTimeFilter[];
```

Reply:
The idea was that it is the list of currently excluded class times, but I agree it makes sense to change :white_check_mark:

Reply:
A few other places were `excluded...` is used. Both are completely fine, but we should probably be consistent 🤗

Reply:
Yeah, working on it... Just pushed what I had time to the other day, didn't even consider whether CI would run at the time... Build errors are fixed now, working on implementing Grid layout...

Reply:
Should be complete now, with `isClassExcludedByTimeFilter` remaining as `excluded` as it is not directly refering to the type but its function...